ci-operator/config/openshift-logging/interop-ocp-ci/openshift-logging-interop-ocp-ci-main__hypershift-4.14-lp-rosa-hypershift.yaml

```diff
@@ -0,0 +1,118 @@
+base_images:
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.14"
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: cluster-logging-operator-rosa-hypershift
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLO_SUB_CHANNEL: stable-5.8
+      CLO_SUB_SOURCE: redhat-operators
+      E2E_RUN_TAGS: '@level0'
+      EO_SUB_CHANNEL: stable
+      EO_SUB_SOURCE: redhat-operators
+      FIREWATCH_CONFIG: |
```

Comment:
@shakyav you may want to make use of the new firewatch functionality here. I will ping you on Slack with more info.